galgebra/mv.py

```diff
@@ -2180,13 +2178,8 @@ def __eq__(self, dop):
             if self.Ga != dop.Ga:
                 return False
 
-            self = Sdop.consolidate_coefs(self)
-            dop = Sdop.consolidate_coefs(dop)
-            if len(self.terms) != len(dop.terms):
-                return False
-            if set(self.terms) != set(dop.terms):
```

Comment:
This crashes because `Mv` objects are not hashable. We should probably fix that, now that they're immutable.